### ISSUE TITLE
Minor issue on split pace

### ISSUE THREAD

**Issue by muktihari** [openivity/openivity.github.io#61]
- Expected last splits 0.03  but got -5.0. I am thinking that maybe we can just omit it if it's below 0.1.
- Do you have plan to add HR here?

<img width="437" alt="image" src="https://github.com/openivity/openivity.github.io/assets/15964188/4e607e4f-5775-4b58-b1b1-4004a33ebe87">

---

- Both HR Zone and Splits Pace share same collapse `id` I guess.

https://github.com/openivity/openivity.github.io/assets/15964188/ba79a47c-069a-4dcb-aa4e-d6db9ffcc2c7

**Comment by raditzlawliet** [comment]
> * Expected last splits 0.03  but got -5.0. I am thinking that maybe we can just omit it if it's below 0.1.
Is split will able to split by meter in the future? 
if there isn't split by meter, omit < 0.1 is nice.

> * Do you have plan to add HR here?
Not yet, 
What idea to display HR here? Average HR of split?
If HR not available, what will it display?

Because there's also request, i'll add this as bug+improvement

**Comment by muktihari** [comment]
> Is split will able to split by meter in the future?
> if there isn't split by meter, omit < 0.1 is nice.

No, split by meters doesn't really matter. We commonly think at least per 1km / 1 miles

> What idea to display HR here? Average HR of split?

Yes, Avg HR of that split. We can add time inside the bar if that seems ok, if not then skip it (?)

**Comment by raditzlawliet** [comment]
The issue Split Pace minus still occure, so i'll continue on this with new PR

**Comment by muktihari** [comment]
> I see, i have found a last record without distance, causing distance into minus.
> I'll adjust it as invalid if distance is invalid

Yes, all fields are not guaranteed to present in a record, especially fit files exported from Strava, even though I have normalised it, still 1-2 records will just contain few or even 1 just a single field.

> Btw, is record.distance always guaranteed increased? Can it use as current total distance in each record?

Yes, it always increasing.

**Comment by raditzlawliet** [comment]
I can make it start from 116 (by flag the start as first distance), 
but i think there will be another issue when using multiple session with more different sport (run, cycing, run)

Or flag it as new split on each new session

**Comment by raditzlawliet** [comment]
> * figure 3 it looks like you merge two session's records? if different session, can we start new line? let's splits should end per session.

not merged, i'm using your file Ride (which contain combined multiple session)

**Comment by muktihari** [comment]
You can create separate PR for this if you want

**Comment by muktihari** [comment]
Nice, so it will adjust as at which distance is the eligible session for splits is starting, right?

**Comment by raditzlawliet** [comment]
> Nice, so it will adjust as at which distance is the eligible session for splits is starting, right?

Yes, it will adjust from `record.distance` accordingly
